subprojects/language-java/src/testFixtures/groovy/org/gradle/language/fixtures/IncrementalAnnotationProcessorFixture.groovy

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.language.fixtures
+
+import org.gradle.test.fixtures.file.TestFile
+
+/**
+ * Generates the source for an Incap-enabled annotation processor that handles an
+ * `@Incremental` annotation.  For classes annotated with `@Incremental`, the processor
+ * generates a <class-name>Incremental.class.
+ */
+class IncrementalAnnotationProcessorFixture {
+
+    // TODO:  Can we query this somehow, from our actual project dependencies?
+    final String INCAP_VERSION = 'org.gradle.incap:incap-ap-client:0.0.7-SNAPSHOT'
```

Comment:
This is making the tests fragile, we should definitely used bintray artifact to ensure that the incap version used is not volatile.